tests/README.md

```diff
@@ -37,3 +37,45 @@ MOONBEAM_LOG="warn,rpc=trace" npm run test
 
 The test script will find available ports above 20000 in order to ensure that it doesn't conflict
 with any other running services.
+
+# Debugging a Moonbeam node
+
+The repository contains a pre-configured debugger configuration for VSCode with the **CodeLLDB**
+(`vadimcn.vscode-lldb`) extension.
+
+Before debugging, you need to build the node with debug symbols enabled by running
+`./scripts/debugger-build.sh`. Then go in the **Debug** tab in the left bar of VSCode and make sure
+**Launch Moonbeam Node (Linux)** is selected in the top dropdown.
+
+To launch the debug session click on the green "play" arrow next to the dropdown. It will take some
+time before the node starts, but the terminal containing the node output will appear when it is
+really starting.
+
```

Comment:
What do I do once the node started? Run `npm run test-single` ?

Reply:
You can do anything you can do with a node. Send transactions to it, perform RPC calls, run a single test with with `DEBUG_MODE=true` ...

Reply:
So maybe add soemthing like "The node is now listening on wsPort = 19933 and  rpcPort = 19932. Run `npm run test-debug` to run a test against it"